restricted_site_access.php

```diff
@@ -748,12 +748,9 @@ public static function enqueue_settings_script() {
 			return;
 		}
 
-		$min    = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
-		$folder = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? 'src/' : '';
-
 		wp_enqueue_script(
 			'rsa-settings',
-			plugin_dir_url( __FILE__ ) . 'assets/js/' . $folder . 'settings' . $min . '.js',
+			plugin_dir_url( __FILE__ ) . 'assets/js/build/settings.min.js',
 			array( 'jquery-effects-shake' ),
```

Comment:
Webpack config could be improved with DependencyInjection plugin. It will help to manage dependencies and versions in wp_enqueue_script calls.